documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc

```diff
@@ -6,25 +6,39 @@
 [id='deploying-the-topic-operator-using-the-cluster-operator-{context}']
 = Deploying the Topic Operator using the Cluster Operator
 
+This procedure describes how to deploy the Topic Operator using the Cluster Operator. Alternatively, you can deploy the Topic Operator as a standalone component. For more information, see xref:deploying-the-topic-operator-standalone[].
+
 .Prerequisites
 
 * A running Cluster Operator
 * A `Kafka` resource to be created or updated
 
 .Procedure
 
-. Topic Operator can be included in the Entity Operator.
-Edit the `Kafka` resource ensuring it has a `Kafka.spec.entityOperator` object that configures the Entity Operator.
-
+. Ensure that the `Kafka.spec.entityOperator` object exists in the `Kafka` resource. This configures the Entity Operator.
++
+[source]
```

Comment:
Add YAML as a format for this source block

Reply:
I don't think AsciiDoctor supports syntax highlighting for YAML. Given the time constraints, I suggest we merge this as-is and then investigate further.

Reply:
OK, I see how this done.